jwst/tweakreg/wcsimage.py

```diff
@@ -592,17 +592,23 @@ def _calc_chip_bounding_polygon(self, stepsize=None):
         if self.wcs is None:
             return
 
-        ny, nx = self.imshape
+        if self.wcs.bounding_box is None:
+            lx = -0.5
```

Comment:
You should use the existing function that does this in
```
from jwst.assign_wcs.util import bounding_box_from_shape

self.wcs.bounding_box = bounding_box_from_shape(self.imshape)
```

Reply:
Do note that you are in a race with @sosey, as she has a PR that will change the name of this function in #3014.  So maybe wait to for her to merge before this is done.

Reply:
muahahahahaha

Reply:
😈

Reply:
so, as James mentions above, are you going to change this section to use the bounding_box_xxx function or do you want to merge this and then change it?

Reply:
@sosey I am not familiar with `assign_wcs` but based on https://github.com/spacetelescope/jwst/pull/3014/files#diff-0173fbd4fd4061722ffb9856bf96c4e9R194 I have trouble seeing how is anything affecting `bounding_box` used here. It looks like it is still going to be in place... right?